Add new analytics

cmcewen opened the pull request:
### Description

Add new segment integration

cmcewen committed:
Add new analytics

annakaz reviewed (approved):
Thanks for fixing this

annakaz commented on packages/mobile/android/app/src/main/AndroidManifest.xml:
```diff
@@ -12,7 +12,7 @@
   <uses-sdk android:minSdkVersion="16" />
 
-  <application android:name=".MainApplication" android:allowBackup="true" android:fullBackupContent="@xml/backup_rules" android:icon="@mipmap/ic_launcher" android:roundIcon="@mipmap/ic_launcher_round" android:label="@string/app_name" android:theme="@style/AppTheme" android:networkSecurityConfig="@xml/network_security_config">
+  <application android:name=".MainApplication" android:allowBackup="false" android:icon="@mipmap/ic_launcher" android:roundIcon="@mipmap/ic_launcher_round" android:label="@string/app_name" android:theme="@style/AppTheme" android:networkSecurityConfig="@xml/network_security_config">
```
How is backup related to this PR?

cmcewen replied:
I ran into an issue with the manifest merging, and it's actually something we should have already disabled for security reasons

mergify[bot] committed:
Merge branch 'master' into cmcewen/analytics-af

mergify[bot] committed:
Merge branch 'master' into cmcewen/analytics-af

cmcewen committed:
update pods

cmcewen committed:
Merge branch 'cmcewen/analytics-af' of github.com:celo-org/celo-monorepo into cmcewen/analytics-af

mergify[bot] committed:
Merge branch 'master' into cmcewen/analytics-af

mergify[bot] committed:
Merge branch 'master' into cmcewen/analytics-af

jeanregisser reviewed (approved):
Looks good!
Why AppsFlyer specifically?

jeanregisser committed:
Patch for segment analytics build on iOS

Remove once these PRs land in a release:
- https://github.com/segmentio/analytics-react-native/pull/234
- https://github.com/segmentio/analytics-react-native/pull/240
